bangc-ops/mlu_op.h

```diff
@@ -2558,6 +2558,111 @@ mluOpStatus_t MLUOP_WIN_API mluOpYoloBox(
+ * ::mluOpTensorDescriptor_t.
+ * @param[in] points
+ * Pointer to the MLU memory that stores the input features' tensor. The points'
+ * in shape (B, C, M), B is batch size, C is channel size, M is number of
+ * elements in one input feature map.
+ * @param[in] idx_desc
+ * The descriptor of the idx tensors. For detailed information, see
+ * ::mluOpTensorDescriptor_t.
+ * @param[in] idx
+ * Pointer to the MLU memory that stores the input indicies' tensor. The idx's
+ * in shape (B, N, 3), B is batch size, C is channel size, N is number of
+ * elements in one output feature map.
+ * @param[in] weight_desc
+ * The descriptor of the weight tensors. For detailed information, see
+ * ::mluOpTensorDescriptor_t.
+ * @param[in] weight
+ * Pointer to the MLU memory that stores the input weight's tensor. The weight's
+ * in shape (B, N, 3), B is batch size, C is channel size, N is number of
+ * elements in one output feature map.
+ * @param[in] b
+ * The size of batch.
+ * @param[in] c
+ * The size of channel.
+ * @param[in] m
+ * The size of one input feature map's elements' number.
+ * @param[in] n
+ * The size of one output feature map's elements' number.
+ * @param[in] output_desc
+ * The descriptor of the output tensors. For detailed information, see
+ * ::mluOpTensorDescriptor_t.
+ * @param[out] output
+ * Pointer to the MLU memory that stores the output features' tensor. The
+ * output's shape is (B, C, N), B is batch size, C is channel size, N is number
+ * of elements in one output feature map.
+ *
+ * @par Return
+ * - ::MLUOP_STATUS_SUCCESS, ::MLUOP_STATUS_BAD_PARAM,
+ *   ::MLUOP_STATUS_NOT_SUPPORTED
+ *
+ * @par Data Type
+ * - The supported data types of input and output tensors are as follows:
+ *   - points tensor: half, float.
+ *   - idx tensor: int.
+ *   - weight tensor: half, float.
+ *   - output tensor: half, float.
+ *   - b: int.
+ *   - c: int.
+ *   - m: int.
+ *   - n: int.
+ *
+ *  @par Data Layout
+ *  - The supported data layout of \b points, \b idx, \b weight, \b output are
+ *    as follows:
+ *
+ *   - points tensor: \p MLUOP_LAYOUT_ARRAY.
+ *   - idx tensor: \p MLUOP_LAYOUT_ARRAY.
+ *   - weight tensor: \p MLUOP_LAYOUT_ARRAY.
+ *   - output tensor: \p MLUOP_LAYOUT_ARRAY.
+ *
+ *  @par Scale Limitation
+ *  - The dimension of \b points should be equal to 3.
+ *  - The dimension of \b idx should be equal to 3.
+ *  - The dimension of \b weight should be equal to 3.
+ *  - The dimension of \b output should be equal to 3.
+ *  - The dimension of \b b should be equal to 1.
+ *  - The dimension of \b c should be equal to 1.
+ *  - The dimension of \b m should be equal to 1.
+ *  - The dimension of \b n should be equal to 1.
+ *
+ * @par Requirements
+ * - None.
+ *
+ *  @par Note
+ *  - The value of \b idx must be in range of [0, M-1], or the output result is
+ *    meaningless and the corresponding output will be set to 0.
+ *  - In MLU270 and MLU290, the max value in the \b idx should be less than
```

Comment:
max > maximum

or the output result is not guaranteed to be correct. > otherwise, the output result may be incorrect.

Reply:
done